src/sentry/static/sentry/app/views/organizationDiscover/result/chart.jsx

```diff
@@ -64,21 +65,48 @@ export default class Result extends React.Component {
         });
       }
 
-      result.push({seriesName: key, data: output[key].data});
+      let seriesNameValue = key;
+      if (key.length > 60) {
+        seriesNameValue = key.substring(0, 60) + '...';
+      }
+      result.push({seriesName: seriesNameValue, data: output[key].data});
     }
     return result;
   }
 
+  checkLineChart(chartData) {
+    for (let i = 0; i < chartData.length; i++) {
```

Comment:
This is kind of weird nesting a functional check inside of a `for` loop, I would use a combination of `find` with `filter` instead.